### PR TITLE
Niigata runs=4L

### PR DESCRIPTION
**Pull request by reiy24**
Still a bit of a result with functioning_results (it is enough to run 03_visualization.Rmd) I've already verified that all/the vast majority of the plans are valid... So check_valid() not working is confusing.

### CODE REVIEW

**Comment by kosukeimai** [comment]
@reiy24 If this is a question about `redist`, please ask at Slack

File: analyses/15_niigata/02_sim_15_niigata.R [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+# 新潟市北区 has an enclave within 阿賀野氏, so we merge the two municipalities.
+pref_enclave_1 <- pref %>%
+  dplyr::filter(code %in% c(15101, 15223)) %>%
+  dplyr::summarise(code = code,
+                   pop = pop,
+                   geometry = sf::st_union(geometry),
+                   gun_code = gun_code,
+                   merged_code = 15901)
+
+# 刈羽村 consists of two discontiguous parts within 柏崎市, so we merge the two municipalities
```
Comment (shomiyazaki) [inline analyses/15_niigata/02_sim_15_niigata.R:23]: According to Wikipedia (https://ja.wikipedia.org/wiki/飛地#日本), 刈羽村 is also surrounded by 長岡市.

File: analyses/15_niigata/02_sim_15_niigata.R [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+pref_enclave_2 <- pref %>%
+  dplyr::filter(code %in% c(15205, 15504)) %>%
+  dplyr::summarise(code = code,
+                   pop = pop,
+                   geometry = sf::st_union(geometry),
+                   gun_code = gun_code,
+                   merged_code = 15902)
+
+# 見附市 has an enclave within 長岡市, so we merge the two municipalities.
```
Comment (shomiyazaki) [inline analyses/15_niigata/02_sim_15_niigata.R:32]: According to Wikipedia (https://ja.wikipedia.org/wiki/飛地#日本), this 佐野町 no longer has population. If 佐野町is the only 飛地 and `pop=0` In the laterst 2020 census, I think we can manually deal with (check Hiroshima example) this with documentation. In this way, I think the one of the constraints would be removed.

**Review by shomiyazaki — COMMENTED** [review]
Hi @reiy24 . I checked your codes and made some comments. Your problem is probably caused by 飛地. It would be great if you check and try my comments.

**Pull request closed without merging**